tripal_chado/src/Plugin/TripalBackendPublish/ChadoPublish.php

```diff
@@ -916,6 +972,33 @@ protected function insertEntities($matches, $titles) {
     }
   }
 
+  /**
+   * Performs bulk deletion of existing entities from the tripal_entity table.
+   *
+   * @param array $entity_ids
+   *   A list of entities to delete.
+   *
+   * @return int
+   *   The number of entities deleted.
+   */
+  protected function deleteEntities(array $entity_ids): int {
+    $storage = \Drupal::entityTypeManager()->getStorage('tripal_entity');
```

Comment:
It looks like we don't actually use the storage here? Perhaps this line can be removed entirely.